models/news.models.js

```diff
@@ -25,3 +35,19 @@ exports.selectArticleCommentsById = (article_id) => {
 		return comments.rows
 	})
 }
+
+exports.updateArticleVotes = (article_id, body) => {
```

Comment:
Have these models `updateArticleVotes` and `selectUsers` been tested? 
Not sure whether you have some code missing, maybe from another branch?